sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -58,11 +58,12 @@ case class ScriptTransformation(
     child.execute().mapPartitions { iter =>
       val cmd = List("/bin/bash", "-c", script)
       val builder = new ProcessBuilder(cmd)
+      builder.redirectErrorStream(true)
```

Comment:
Isn't this pushing stderr into stdout? that mixes error output with the program output.
I'm not arguing against consuming stderr, but maybe against pouring into stdout.

BTW is this still an issue with `ProcessBuilder`? I know it was a problem in Java back in the day; you had to consume the streams or face deadlock. I imagine it still is but I haven't written code for `ProcessBuilder` in a long time.